src/blocks/author/index.js

```diff
@@ -28,6 +28,13 @@ const settings = {
 	description: __( 'Add an author biography to build credibility and authority.' ),
 	icon,
 	keywords: [ __( 'biography' ), __( 'profile' ), 'coblocks' ],
+	example: {
+		attributes: {
+			name: 'Jane Doe',
+			biography: __( 'Born to express, not to impress. A maker making the world I want.' ),
+			imgUrl: '/wp-content/plugins/coblocks/dist/images/examples/author.jpg',
```

Comment:
This might break if the user has specified a different wp-content folder.

Reply:
Unless base URL is properly set in the head of HTML document.

Reply:
Good call. Looks like we need to expose the url produced through [`plugins_url()`](https://developer.wordpress.org/reference/functions/plugins_url/) in the admin to create the proper URL here.

Reply:
This is how webpack can handle public paths:

https://webpack.js.org/guides/public-path/

If you will find it useful, we can add it to wp-scripts build, too.

Reply:
I'm still planning to figure this out so we can server images locally. For now we're working on using an external source for now in #1106.